client/src/modules/Registry/constants/index.ts

```diff
@@ -1,20 +1,79 @@
-export const TECHNOLOGIES = [
-  { id: 'nodejs', displayName: 'Node.js' },
-  { id: 'react', displayName: 'React' },
-  { id: 'angular', displayName: 'Angular' },
-  { id: 'ios', displayName: 'IOS' },
-  { id: 'android', displayName: 'Android' },
-];
+import { Rule } from 'antd/lib/form';
 
 export const DEFAULT_COLUMN_SIZES = { xs: 24, sm: 24, md: 24, lg: 24, xl: 24 };
 export const DEFAULT_DOUBLE_COLUMN_SIZES = { xs: 24, sm: 24, md: 12, lg: 12, xl: 12 };
 export const DEFAULT_ROW_GUTTER = 24;
 
+export const TEXT_EMAIL_TOOLTIP = 'No spam e-mails. Only for course purposes.';
 export const TEXT_LOCATION_STUDENT_TOOLTIP =
   'We need your location for understanding the audience and for mentor distribution.';
-export const TEXT_LOCATION_MENTOR_TOOLTIP =
-  'We need your location for understanding audience and use it for students distribution';
-export const TEXT_EMAIL_TOOLTIP = 'No spam e-mails. Only for course purposes.';
-export const TEXT_EPAM_EMAIL_TOOLTIP =
-  'If you are EPAM employee, please specify your email to avoid some manual processes later';
 export const RSSCHOOL_BOT_LINK = 'https://t.me/rsschool_bot?start';
+export const DATA_PROCESSING_TEXT =
+  'I agree to the processing of my personal data contained in the application and sharing it with companies only for students employment purposes.';
+export const SUCCESS_TEXT =
+  'Before you start we need to consider your application and submit you to a course. It could take some time. We will send you next steps via an email on the address you provided during registration.';
+
+export const ERROR_MESSAGES = {
+  chooseAtLeastOne: 'Should choose at least one',
+  shouldAgree: 'Should agree to the data processing',
+  inEnglish: (prop: string) => `${prop} should be in English`,
+  email: 'Invalid email',
+  epamEmail: 'Please enter a valid EPAM email',
+  location: 'Please select city',
+  phone: 'Invalid phone number',
+  tryLater: 'An error occurred. Please try later',
+};
+
+export const TOOLTIPS = {
+  locationMentor: 'We need your location for understanding audience and use it for students distribution',
+  primaryEmail: 'No spam e-mails. Only for course purposes.',
+  epamEmail: 'If you are EPAM employee, please specify your email to avoid some manual processes later',
+};
+
+export const PLACEHOLDERS = {
+  firstName: 'Dzmitry',
+  lastName: 'Varabei',
```

Comment:
are you sure it's a good idea to use real name and last name as placeholder?
maybe use name and last name text?

Reply:
fixed

Reply:
I'm not sure, It already was there, current form contains that name. I think it somehow related to extra on the first name input: `In English, as in passport` which I don't get, why specifically as in passport?

Reply:
maybe use John Doe ?